_inc/idc-notice.js

```diff
@@ -18,7 +18,11 @@
 
 	// Initialize Tracks and bump stats.
 	if ( 'undefined' !== typeof analytics ) {
-		analytics.initialize( tracksUser.userid, tracksUser.username );
+		analytics.initialize(
```

Comment:
Some searching suggests that this and the `seach-widget-admin` I mentioned in my review actually appear to use [tracks-callables](https://github.com/Automattic/jetpack/blob/master/_inc/lib/tracks/tracks-callables.js), which isn't updated at all for `blogid`.

https://github.com/Automattic/jetpack/blob/c36c5e2847a517a04f291f72f941072a8dc8e343/class.jetpack-idc.php#L305
https://github.com/Automattic/jetpack/blob/c36c5e2847a517a04f291f72f941072a8dc8e343/modules/widgets/search.php#L107

Reply:
great catch @sirreal, we missed that. I'm working on updating the patch.

Reply:
We might investigate why `superProps` was added and whether it's used. I have no idea how it might be used. `getAll` doesn't clarify exactly what is expected to be passed there.